src/components/Setup/SyncScreen.tsx

```diff
@@ -45,13 +45,17 @@ const SyncScreen: FC<InputData> = ({ data, callback }) => {
     setPassword(event.target.value);
   };
 
-  const unlockWallet = async () => {
+  const unlockWallet = () => {
     setRunningUnlock(true);
     setPasswordWrong(false);
-    await instance
+    instance
```

Comment:
If we want to avoid post/then/catch but use async/await I guess The code from before with a try/catch of instance.post could work.  
Not important. I general I think async/await is easier to handle/read.

Reply:
I'm not Sure what's better, since I like the then/catch for small things better than wrapping everything again in a try/Catch.

Will think about it 👍